py3status/modules/backlight.py

```diff
@@ -45,6 +49,10 @@
 
 from __future__ import division
 import os
+try:
+    from pydbus import SystemBus
+except ModuleNotFoundError:
+    None
```

Comment:
nit: `pass` would probably be more natural in this case

Reply:
right... i'm not very good at python. Still learning the coding style...